.github/workflows/build-ci.yml

```diff
@@ -434,12 +401,19 @@ jobs:
         uses: burrunan/gradle-cache-action@v1
         with:
           job-id: build-envoy
-          arguments: --scan envoy:prepareDocker
+          arguments: --scan outputVersion envoy:prepareDocker
           gradle-version: wrapper
 
+      - name: Get Deephaven Version
+        id: deephaven_version
```

Comment:
Do you need this again? The deephaven_version action is already above, doing the same thing...

Reply:
It's in a separate job